src/attr/_make.py

```diff
@@ -229,13 +229,13 @@ class MyClassAttributes(tuple):
 
 
 def _is_class_var(annot):
-    """
-    Check whether *annot* is a typing.ClassVar.
+    """Check whether *annot* is a typing.ClassVar.
 
-    The implementation is gross but importing `typing` is slow and there are
-    discussions to remove it from the stdlib alltogether.
+    The string comparison hack is used to avoid evaluating all string
+    annotations which would put attrs-based classes at a performance
+    disadvantage compared to plain old classes.
     """
-    return str(annot).startswith("typing.ClassVar")
+    return str(annot).startswith(("typing.ClassVar", "t.ClassVar", "ClassVar"))
```

Comment:
I think we should amend our tests to cover the new cases?

Reply:
Yes.